diff --git a/cps/__init__.py b/cps/__init__.py
index 269e4aca..e2b828a9 100644
--- a/cps/__init__.py
+++ b/cps/__init__.py
@@ -150,7 +150,6 @@ def create_app():
 
     lm.login_view = 'web.login'
     lm.anonymous_user = ub.Anonymous
-    lm.session_protection = 'strong' if config.config_session == 1 else "basic"
 
     db.CalibreDB.update_config(config)
     db.CalibreDB.setup_db(config.config_calibre_dir, cli_param.settings_path)
diff --git a/cps/admin.py b/cps/admin.py
index 20c58aac..99d2ad19 100644
--- a/cps/admin.py
+++ b/cps/admin.py
@@ -101,8 +101,6 @@ def admin_required(f):
 
 @admi.before_app_request
 def before_request():
-    if not ub.check_user_session(current_user.id, flask_session.get('_id')) and 'opds' not in request.path:
-        logout_user()
     g.constants = constants
     g.google_site_verification = os.getenv('GOOGLE_SITE_VERIFICATION','')
     g.allow_registration = config.config_public_reg
