diff --git a/lua/kitty-scrollback/health.lua b/lua/kitty-scrollback/health.lua
index 9d87dfa..a231b8d 100644
--- a/lua/kitty-scrollback/health.lua
+++ b/lua/kitty-scrollback/health.lua
@@ -234,7 +234,7 @@ local function check_kitty_debug_config()
 end
 
 local function check_kitty_scrollback_nvim_version()
-  local current_version = nil
+  local current_version = 'master'
   local tag_cmd = { 'git', 'describe', '--exact-match', '--tags' }
   local ksb_dir =
     vim.fn.fnamemodify(vim.api.nvim_get_runtime_file('lua/kitty-scrollback', false)[1], ':h:h')
